tequilapi/endpoints/connection.go

```diff
@@ -196,7 +196,7 @@ func (ce *ConnectionEndpoint) Create(resp http.ResponseWriter, req *http.Request
 		return
 	}
 
-	proposals, err := ce.proposalProvider.FindProposals(cr.ProviderID, cr.ServiceType)
+	proposals, err := ce.proposalProvider.FindProposals(cr.ProviderID, cr.ServiceType, "", "")
```

Comment:
It's error-prone to have 4 unnamed params all of the same type for filtering. Perhaps it's better to have some sort of struct for that?